docs/articles/documentation/extending-testcafe/creating-custom-reporter/reporterpluginhost.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: docs
+title: ReporterPluginHost Object
+permalink: /documentation/extending-testcafe/creating-custom-reporter/reporterpluginhost/
+---
+# ReporterPluginHost Object
+
+The ReporterPluginHost object provides access to a custom reporter scaffolded by the [TestCafe reporter generator](https://github.com/DevExpress/generator-testcafe-reporter).
```

Comment:
Nope. Your reporter should implement some methods. And TestCafe will mixin some helper methods into it.